edison/static/js/policy.controller.js

```diff
@@ -1,9 +1,59 @@
 
 import { Policy } from './policy.model.js'
 import { PolicyElementsModifier } from './policyElementsModifier.js'
+import { PoliciesExampleArray } from './policiesExampleArray.js'
 
 var policy = new Policy();
 var countCondition = 0;
+var policiesExamples = new PoliciesExampleArray();
+
+function setCardBody(policy) {
+    var cardBody = createInitElement('div', 'card-body');
+    var cardTitle = createInitElement('h5', 'card_title');
+    var cardText = createInitElement('h5');
+
+    cardBody.appendChild(cardTitle);
+    cardBody.appendChild(cardText);
+    cardText.innerHTML = `This policy is associated with ${policy.room} room and responsible for ${policy.command}
+    with these conditions: ${policy.condition}`;
+
+    return cardBody;
+}
+
+function createCard(policyNum, policy) {
+    var card = createInitElement('div', 'card');
+    var headerCard = createInitElement('div', 'card-header');
+    var buttonTextSize = createInitElement('h5', 'mb-0');
+    var headerCardButton = createInitElement('button', 'btn btn-link');
+
+    card.appendChild(headerCard);
+    headerCard.appendChild(buttonTextSize);
+    headerCardButton.setAttribute("data-toggle", "collapse");
+    headerCardButton.setAttribute("data-target", "#collapse-policy" + policyNum);
+    headerCardButton.setAttribute("aria-expanded", "true");
+    headerCardButton.setAttribute("aria-controls", "collapse-policy" + policyNum);
+    headerCardButton.innerHTML = policy.name;
+    buttonTextSize.appendChild(headerCardButton);
+
+    return card;
+}
+
+function showPolicies() {
+    //once the route will be ready this function will be modifiy to get the policies from the backend and convert them to array 
+    var len = policiesExamples.examples.length;
+    var showPolicies = document.getElementById('show-policies');
+    for (var i = 0; i < len; i++) {
+        var card = createCard(i, policiesExamples.examples[i]);
+        var collapseClassPolicy = createInitElement("div", "collapse", "collapse-policy" + i);
+        var cardBody = setCardBody(policiesExamples.examples[i]);
```

Comment:
maybe you can build the example class in more generic way (by inheritance or interface), so when have the real policies from BE, you can do the change really quick , and also you won't have unnecessary code in the PR that you know you will have to remove at some point